packages/site/landing/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Nimble</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel="icon" href="favicon.ico" sizes="any">
+    <script type="module" src="index.ts"></script>
+    <link rel="stylesheet" type="text/css" href="./styles.scss" />
+</head>
+
+<body>
+        <div class="container">
+            <div class="main-content">
+                <div class="banner">
+                    <div class="banner-main">
+                        <img src="./images/nimble-logo-black.png" alt="Nimble Logo">
+                        <p>The Nimble Design System is an easy to use collection of well-designed, styled components
+                            that can be used in any NI web application.</p>
+                    </div>
+                </div>
+                 <div class="cards">
+                   <div class="card-wrapper">
+                       <div class="card" href="https://github.com/ni/nimble">
```

Comment:
Didn't try it locally but looks like the anchor tags were accidentally turned into divs, they won't function as links anymore

Reply:
That's on me. Got it resolved. Should be good now. 
https://github.com/ni/nimble/pull/852/commits/956cd689b5e77dfb8002b52bf12a0187aa768362